src/plugins/messageStorage/sqlite.js

```diff
@@ -139,6 +139,10 @@ class MessageStorage {
 			));
 		});
 	}
+
+	canProvideMessages() {
```

Comment:
What does `canProvideMessages` mean?

Reply:
It means this storage can load stored messages back into memory.

Reply:
Oh I see. Kind of a bikeshed but maybe we need better wording for this. `supportsReload`? `supportsRead`?
Or instead, maybe only if `getMessages` is available and `isEnabled` is true or something?